locale/areas.ru.tr

```diff
@@ -0,0 +1,135 @@
+Rename an area that you own=Переименовать территорию, которой вы владеете.
+Self protection is disabled.=Защита своих территорий отключена.
+Self protection is enabled.=Защита своих территорий включена.
+That area doesn't exist.=Территория не существует.
+The player "@1" does not exist.=Игрок "@1" не существует.
+Toggle an area open (anyone can interact) or closed=Открыть/закрыть территорию для других игроков
+Toggle an area open/closed for members in your faction.=Открыть/закрыть территорию для игроков вашей фракции.
+You are an area administrator ("areas" privilege).=Вы можете управлять всеми территориями (привилегия "areas")
+You can protect areas=Вы можете защищать территории
+You can't protect that area.=Вы не можете установить защиту на эту территорию.
+You can't protect that area: @1=Вы не можете установить защиту на территорию: @1
+You don't have the necessary privilege ("@1").=У Вас нет необходимой привилегии ("@1").
+You don't own that area.=Вы не являетесь владельцем этой территории.
+You have @1 areas.=У вас @1 территорий.
+
+You have been granted control over area #@1. Type /list_areas to show your areas.=Вам выдан доступ к территории #@1.
+
+You have extended area protection limits ("areas_high_limit" privilege).=У вас увеличенный лимит на кол-во создаваемых вами защищённых территорий
+
+You have the necessary privilege ("@1").=У вас есть необходимая привилегия ("@1").
+You need to select an area first.=Сначала необходимо выделить территорию.
+
+### chatcommands.lua ###
+### pos.lua ###
+
+<ID>=
+Invalid usage, see /help @1.=Неверное использование, см. /help @1.
+
+### hud.lua ###
+
+:open=:открытая
+Areas:=Территории:
+
+### init.lua ###
+
+Can administer areas.=Может управлять территориями.
+Can protect areas.=Может создавать защищённые территории.
+Can protect more, bigger areas.=Может создавать защищённые территории больше территорий и с бОльшим размером.
+
+### interact.lua ###
+
+@1 is protected by @2.=@1 - принадлежит @2.
+
+### internal.lua ###
+
+Area is too big.=Территория слишком велика.
+
+Self protection is disabled or you do not have the necessary privilege.=Возможность защиты территорий отключена или же Вы не имеете необходимых привилегий.
+
+The area intersects with @1 [@2] (@3).=Территория пересекается с @1 [@2] (@3).
+
+You have reached the maximum amount of areas that you are allowed to protect.=Вы достигли максимально допустимого количества на создание защищённых территорий.
+
+
+### legacy.lua ###
+
+<version>=<версия>
+Converted areas saved. Done.=Готово. Сконвертированные территории сохранены.
+Converting areas…=Конвертирование территорий…
+Error loading legacy file: @1=Ошибка загрузки файла с устаревшим форматом: @1
+Invalid version number. (0 allowed)=Неверный номер версии. (поддерживается 0)
+Legacy file loaded.=Файл с устаревшим форматом загружен.
+
+Loads, converts, and saves the areas from a legacy save file.=Загружает, конвертирует и сохраняет файл с устаревшим форматом.
```

Comment:
your translation `Loads, converts, and saves a legacy file.` (`the areas from` is missing)